trin-core/src/utp/stream.rs

```diff
@@ -60,6 +60,8 @@ const MAX_BASE_DELAY_AGE: Delay = Delay(60_000_000);
 const DISCV5_SOCKET_TIMEOUT: u64 = 25;
 /// uTP receive timeout in milliseconds
 const UTP_RECEIVE_TIMEOUT: u64 = 1000;
+/// Maximum retries trying to receive acknowledgments for the send packets
+const MAX_RECV_ACKS_RETRIES: u8 = 5;
```

Comment:
I don't have a sense of what a good limit is, but is there any empirical evidence for why we choose 5 here?

Reply:
No empirical evidence, Fluffy is using 4 but I'm happy to change this to any reasonable value (< 10).